js/components/videoQualityPanel.js

```diff
@@ -93,7 +101,6 @@ VideoQualityPanel.propTypes = {
   videoQualityOptions: React.PropTypes.shape({
     availableBitrates: React.PropTypes.arrayOf(React.PropTypes.shape({
       id: React.PropTypes.string,
-      bitrate: React.PropTypes.number,
```

Comment:
instead of removing this, change it to -

```
bitrate: React.PropTypes.oneOfType([
    React.PropTypes.string,
    React.PropTypes.number,
  ]),
```